### DISCUSSION TITLE
Is there a way to turn minutes into decimals?

### DISCUSSION THREAD

**Discussion by coolcat818382901** [kimai/kimai#5035] [General]
On exports, and in other places in Kimai, can I turn minutes into decimals?

Example: If I punched in 2 hours and 37 minutes, I don't want the system to say 2:37. I want it to say 2.62

**Comment by kevinpapst** [comment]
Yes, check user preferences

**Reply by coolcat818382901** [reply]
Ok, but will this take effect for **all** users? And through all interfaces within Kimai?

**Reply by kevinpapst** [reply]
No: each user can decide on his own and it switches only the exports.

AFAIK humans have a way better understanding of minutes instead of decimals. 
After reading feature requests for so many years its pretty rare to see a unique new one. 
So congrats on that 😄 

I don't mind if you open a feature request for that. Lets see if more people are interested.

**Comment by coolcat818382901** [comment]
Well, typically I prefer minutes but there’s some instances where I’m
programmatically reading data, that what I wrote doesn’t “understand”
minutes without me implementing additional logic.

I’ll open a feature request when I’m on PC shortly. Thank you 😁

**Reply by kevinpapst** [reply]
If you read programmatically, you can use the JSON API (which always returns date-times / seconds).
If you read exported CSVs, those can already be switched to decimal.

A feature request without good use-case is not a good feature request!
Please think about your requirement and include a use-case that makes sense.
